src/libNode/Node.cpp

```diff
@@ -1855,26 +1863,9 @@ bool Node::ProcessDSGuardNetworkInfoUpdate(const bytes& message,
   LOG_GENERAL(INFO, "Received from lookup " << from);
 
   {
+    // Process and update ds committee network info
     lock_guard<mutex> lock(m_mediator.m_mutexDSCommittee);
     for (const auto& dsguardupdate : vecOfDSGuardUpdateStruct) {
-      // Remove old ds guard IP info from blacklist exclude list
```

Comment:
Are these removals related to this PR?

Reply:
Fixed

Reply:
Didn't find the code change. Do we really need to remove this part in this PR?